CDCgov/prime-reportstream
# 9078 - Added facilities & providers page

This PR ...

As part of the STLT Dashboard, we need to implement the design of the facilities/providers index page. This implementation is strictly visual, and we'll have a separate ticket for the table and hooking it up to the API afterward.

Test Steps:
1. Navigate to RS
2. Select a Receiver
3. Go to /data-dashboard
4. Click on the facilities & providers
5. Visually should match [Figma](https://www.figma.com/file/6mwI5ac6rprACKDzDo4Ady/ReportStream-Workspace-%7C-2023?type=design&node-id=2369-27816&t=sSRJvSYgCUQbL2TO-0)

## Changes
<img width="1775" alt="Screenshot 2023-05-16 at 11 08 23 AM" src="https://github.com/CDCgov/prime-reportstream/assets/102491809/572955ae-6abb-4588-b6d9-6b4092a1fd83">


## Checklist

### Testing


## Linked Issues
- Fixes #9078 

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #9507
